### PR TITLE
General rework of optional and excluded page

### PR DESCRIPTION
**Pull request by elharo**
consistent tense and person; assorted cleanups; mention banneddependencies rule

### CODE REVIEW

**Comment by elharo** [comment]
Ping. Any thoughts.

**Comment by hboutemy** [comment]
this one is more complex, I'll need some time to review...
thanks for your help and patience

File: content/apt/guides/introduction/introduction-to-optional-and-excludes-dependencies.apt [hunk 1]
```diff
@@ -195,8 +195,8 @@ B, C, D, E, F
 </dependency>
 +---+
 
-  HOWEVER, they didn't. As a last resort, you still have the option to exclude it 
-  on your side, in Project-A, like this:
+  Unfortunately, they didn't. As a last resort, you can exclude it 
+  on your your own POM for Project-A like this:
```
Comment (nicokosi) [inline content/apt/guides/introduction/introduction-to-optional-and-excludes-dependencies.apt]:
```suggestion
   on your own POM for Project-A like this:
```

File: content/apt/guides/introduction/introduction-to-optional-and-excludes-dependencies.apt [hunk 2]
```diff
@@ -224,9 +224,9 @@ B, C, D, E, F
 
 +---+
 
-  If we deploy the Project-A to a repository, and Project-X declares a 
-  normal dependency on Project-A, will Project-D be excluded from the 
-  classpath still?
+  If you deploy  Project-A to a repository, and Project-X declares a
```
Comment (nicokosi) [inline content/apt/guides/introduction/introduction-to-optional-and-excludes-dependencies.apt]:
```suggestion
  If you deploy Project-A to a repository, and Project-X declares a 
```

**Comment by elharo** [comment]
Any interest in pursuing this one?

**Comment by michael-o** [comment]
@hboutemy do you want to continue on this?

**Comment by hboutemy** [comment]
reviewed and merged, thank you

**Pull request closed without merging**